src/utils/index.ts

```diff
@@ -9,7 +9,10 @@ export * from './selectedCellUtils';
 export * from './styleUtils';
 
 export const { min, max, round, floor, sign, abs } = Math;
-export const { escape: cssEscape } = CSS;
```

Comment:
I had done this to skip the property access on the `CSS` object,
so instead of creating a `cssEscape` function we can just use `CSS.escape()` 🤷‍♂️

Reply:
🤦‍♂️